src/main/java/no/nav/veilarbaktivitet/aktivitet/AktivitetDAO.java

```diff
@@ -505,4 +505,26 @@ public void insertLestAvBrukerTidspunkt(long aktivitetId) {
                 WHERE aktivitet_id = :aktivitetId
                 """, params);
     }
+
+    //skal slettes etter patch
+    public void patchKanIkkeEndres(AktivitetData gammelAktivitet) {
+        var params = new MapSqlParameterSource().addValue("aktivitetId", gammelAktivitet.getId());
+
+        namedParameterJdbcTemplate.update("""
+                UPDATE AKTIVITET SET historisk_dato = null, livslopstatus_kode = 'GJENNOMFORES'
```

Comment:
Vi har en mekanisme for å opprette eksternaktiviteter som historisk, så hvis aktiviteten er historisk, bør du sette eksternaktivitet.opprettet_som_historisk og eksternaktivitet.historisk_dato, slik at vi får satt den korrekt til historisk etter på.